src/components/Nav.tsx

```diff
@@ -31,15 +35,39 @@ export function Nav(
     labels: { labelPrevious, labelNext }
   } = useDayPicker();
 
+  const handleNextClick = useCallback(
+    (e: React.MouseEvent<HTMLButtonElement>) => {
+      if (nextMonth) {
+        onNextClick?.(e);
+      } else {
+        e.preventDefault();
+        e.stopPropagation();
+      }
+    },
+    [nextMonth, onNextClick]
+  );
+
+  const handlePreviousClick = useCallback(
+    (e: React.MouseEvent<HTMLButtonElement>) => {
+      if (previousMonth) {
+        onPreviousClick?.(e);
+      } else {
+        e.preventDefault();
+        e.stopPropagation();
```

Comment:
I think it's better internally only ignore the button click and let it bubble up, meaning not stop its propagation and default behavior, so any developer doesn't lose such events.
E.g.: a developer renders a custom button which has a tooltip saying the day picker reached the maximum allowed months, they may need this event to propagate to show the tooltip.

Reply:
Agree! Thanks for the heads up!